### DISCUSSION TITLE
No function with webinterface buttons

### DISCUSSION THREAD

**Discussion by r4pt0r1337** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#740] [Q&A]
I got a porblem with the buttons on the web site. I have updated from an very old version. I think 2 years old to the actual relaease.
But in the release I had issues with some files that are not loades correctly so it looks not good on the page. I found some other issues for this in the Discussions and installed the version 4.2.4. But there i got an other issue i cant't solve till now.

All Buttons in the main page have no function. When i set another temperature and relaod the page it will be reset to the last setting. When I reset the filter timer, nothing happend. Later when i look in the SPA config i see many tasks in the queue, but nothing happened.
When i use the display of the pump all functions work fine.
When i use mqtt also everything works fine.
My Pool ist a Miami from 2021 with the 6 wire layout and the grey edged pump.

As I teared down the pool last fall everything worked fine with the old firmware. But i don't know the version.
I have build a new PCB with another ESP but the same issue happened.

Have anybody an idea?

**Comment by visualapproach** [comment]
Is the time correct on your devices? If the computer sending the commands is in the future it will take some time until the command is executed

**Comment by r4pt0r1337** [comment]
thanks for your fast answer. No the time was in the 1970ties. I have entered a valid NTP server and synced the time.
Now it works fine.
Thank you
